### PR TITLE
fix default output not using realpath

### LINKED ISSUE #68: Error [WinError 5] Access is denied './/output'
When you run Gui.py from a folder outside of the directory (e.g. File Explorer's recent files), it tries to create the output directory there instead of in the install directory. It should default to the `install_dir/output`.

### PR DESCRIPTION
**Pull request by flagrama**
Should fix #68

### CODE REVIEW

**Comment by dpeekstok** [comment]
I've tested this solution and it worked. It correctly found OoT-Randomizer/Ouput and put the generated rom there without requiring input.

**Comment by flagrama** [comment]
I'm not sure if it'd be better to call `local_path()` in `default_output_path()` or not. I'm not sure how this default output path would be handled in an installer like 1.0 or 2.0 either.

**Comment by flagrama** [comment]
image
A LocalRom isn't generated until after the output directory is set.

**Comment by TestRunnerSRL** [comment]
Hmm, I dunno windows is dumb I guess. The thing I'm unsure about is how to handle static builds. It might just have to be you can't do anything about it, not sure.

**Comment by flagrama** [comment]
Is there any major problems with what AA did for his releases? Just do that if bundled and output directory path is `''` and if not bundled or output directory path not `''` just use that path?

**Comment by TestRunnerSRL** [comment]
That statement was logically hard to follow lol. But yeah, I didn't like things being dumped in the running directory. Either way the problem this is trying to fix will still occur if you run from quick access and don't set an output dir (for the bin release) and there's no way to fix that unless you can get the directory of the exe. I wonder if there some of the freeze programs provide a way to get that

**Comment by TestRunnerSRL** [comment]
well i guess I should do one of these I guess https://stackoverflow.com/questions/404744/determining-application-path-in-a-python-exe-generated-by-pyinstaller

**Comment by flagrama** [comment]
Yeah it looks like I wasn't exactly clear... I mean this:
```python
if is_bundled and path == '':
    # Set to oot-randomizer folder inside documents like releases currently do
    # This should already deal with the issue since it isn't using the cwd and is explicitly setting documents path
if path == '':
    # Set path based on realpath
# Else just use the supplied path
```

**Comment by flagrama** [comment]
Also looks like AA's already did the check in his `is_bundled()` function:
```python
def is_bundled():
    return getattr(sys, 'frozen', False)
```
So looks like most of the static build issues should already be handled by reusing some of that code that was previously removed once more.
